quarkus/defaults/src/main/resources/application.properties

```diff
@@ -123,6 +123,7 @@ polaris.persistence.type=in-memory
 polaris.secrets-manager.type=in-memory
 
 polaris.file-io.type=default
+polaris.file-io.default-config.allow-hadoop-file-io=false
```

Comment:
I think we should default to `false` here, which means we'd have to change tests to use a nondefault FileIOFactory which I have not implemented here